src/types/index.ts

```diff
@@ -65,6 +66,7 @@ export interface VacationData {
   type: VacationType;
   message: string;
   days: number;
+  status: VacationRequestStatus[];
```

Comment:
Could we not actually remove this VacationData interface and use the VacationRequest type from generated files based on spec. At a 5 second glance it looks like only major-ish change would be to also be having userId set where we were using this previously, but would that be a problem?